src/ray/raylet/node_manager.h

```diff
@@ -518,12 +530,16 @@ class NodeManager : public rpc::NodeManagerServiceHandler {
   /// The RPC service.
   rpc::NodeManagerGrpcService node_manager_service_;
 
-  /// The `ClientCallManager` object that is shared by all `NodeManagerClient`s.
+  /// The `ClientCallManager` object that is shared by all `NodeManagerClient`s
+  /// as well as all `WorkerTaskClient`s.
   rpc::ClientCallManager client_call_manager_;
 
   /// Map from node ids to clients of the remote node managers.
   std::unordered_map<ClientID, std::unique_ptr<rpc::NodeManagerClient>>
       remote_node_manager_clients_;
+
+  /// Map from worker ids to rpc clients of workers.
+  std::unordered_map<ClientID, std::unique_ptr<rpc::WorkerTaskClient>> worker_clients_;
```

Comment:
This should go in `WorkerPool`, not `NodeManager`.

Reply:
ok, updated.